contracts/Clarinet.toml

```diff
@@ -7,28 +7,28 @@ cache_dir = './.cache'
 requirements = []
 [contracts.sbtc-bootstrap-signers]
 path = 'contracts/sbtc-bootstrap-signers.clar'
-clarity_version = 2
-epoch = 2.5
+clarity_version = 3
+epoch = 3.0
 
 [contracts.sbtc-deposit]
 path = 'contracts/sbtc-deposit.clar'
-clarity_version = 2
-epoch = 2.5
+clarity_version = 3
+epoch = 3.0
 
 [contracts.sbtc-registry]
 path = 'contracts/sbtc-registry.clar'
-clarity_version = 2
-epoch = 2.5
+clarity_version = 3
+epoch = 3.0
```

Comment:
Oh interesting, I didn't realize that clarity v3 was a thing. I didn't see any mention of it in the docs, where can I find out more about this? (Well, I've only looked here https://docs.stacks.co/reference/functions for info in clarity v3)

Reply:
Yeah it's pretty much just an update to include some nakamoto-specific functions like for tenures etc. Don't think the docs are published yet but they will be generated from here:

https://github.com/stacks-network/stacks-core/tree/develop/clarity%2Fsrc%2Fvm%2Fdocs